pandas/tests/test_categorical.py

```diff
@@ -1554,18 +1554,6 @@ def test_deprecated_labels(self):
             res = cat.labels
         self.assert_numpy_array_equal(res, exp)
 
-    def test_deprecated_levels(self):
```

Comment:
yes that IS what ``test_pickles`` does. However a ``Categorical`` that has a levels attribute does not exist in old pickles.